Sources/RemindersLibrary/Reminders.swift

```diff
@@ -74,12 +74,17 @@ public final class Reminders {
         semaphore.wait()
     }
 
-    func addReminder(string: String, toListNamed name: String, dueDate: DateComponents?) {
+    func addReminder(string: String, toListNamed name: String, dueDate: DateComponents?, priority: Priority?) {
         let calendar = self.calendar(withName: name)
         let reminder = EKReminder(eventStore: Store)
         reminder.calendar = calendar
         reminder.title = string
         reminder.dueDateComponents = dueDate
+        switch priority {
```

Comment:
Suggest moving this switch expression to `Priority` as a computed property `value` or something others.